.github/actions/build_pandas/action.yml

```diff
@@ -3,15 +3,28 @@ description: Rebuilds the C extensions and installs pandas
 runs:
   using: composite
   steps:
-
     - name: Environment Detail
       run: |
-        conda info
-        conda list
+        if which mamba 2>/dev/null; then
+          mamba info
+          mamba list
+          mamba info | grep -Ei 'environment.+:' | grep -qEiv 'environment.+:.+none'
+        fi
+        python -VV
+        which python
+        which pip
+        pip list
       shell: bash -el {0}
 
     - name: Build Pandas
       run: |
-        python setup.py build_ext -j 2
-        python -m pip install -e . --no-build-isolation --no-use-pep517 --no-index
+        time python setup.py build_ext -v -j $N_JOBS
+        pip install -v -e . --no-build-isolation --no-use-pep517 --no-index
+      shell: bash -el {0}
```

Comment:
How is this shell different to the default?

Reply:
- It is a login shell (important for Conda to work)
- It fails if one of the commands fails, rather than just continuing.